### ISSUE TITLE
Multiselectlistbox setvalue not working 

### ISSUE THREAD

**Issue by weiliang2** [vaadin/vaadin-list-box#94]
Hi,

refering to https://vaadin.com/forum/thread/18577936/multiselectlistbox-setvalue-error

it used to worked in vaadin 14.3.7. But from 14.4.0 onwards it does not work.

Sample project are in the forum.

Would really appreciate some help.

**Comment by vlukashov** [comment]
might be caused by https://github.com/vaadin/vaadin-list-box-flow/pull/93/files as it seems to be the only change to that component between 14.3.7 and 14.4.0

**Comment by tulioag** [comment]
I can reproduce it in 14.5.3, but it seems to be fixed in 14.6.0.rc1. 

@weiliang2 if you would like to test it, you can update the version in your pom file and add the pre-releases repository for dependencies and plugins
```xml

        <repository>
            <id>vaadin-prereleases</id>
            <url>https://maven.vaadin.com/vaadin-prereleases</url>
        </repository>
```
```xml
        <pluginRepository>
            <id>vaadin-prereleases</id>
            <url>https://maven.vaadin.com/vaadin-prereleases</url>
        </pluginRepository>
```

**Comment by weiliang2** [comment]
yes i have tested them with 14.6.0.rc1 and its working again!
